.eslintrc.js

```diff
@@ -3,114 +3,113 @@ const WARNING = 1
 const ERROR = 2
 
 module.exports = {
```

Comment:
Will you please help me by highlighting what's different in this file? This diff is unhelpful.